BALSAMIC/constants/reference.py

```diff
@@ -133,6 +133,30 @@
             "output_file": "delly_exclusion.tsv",
             "output_path": "genome",
         },
+        "delly_mappability": {
+            "url": "https://raw.githubusercontent.com/Clinical-Genomics/reference-files/86aab2d10c5ffc009bc8c68ad077ab7283d8fe06/cancer/references/GRCh38.delly.blacklist.gz",
```

Comment:
```suggestion
            "url": "https://github.com/Clinical-Genomics/reference-files/blob/master/cancer/references/GRCh37.delly.blacklist.gz?raw=true",
```

Instead of this long commit, maybe you can use links pointing to master